### PR TITLE
Feature/pleskunruslan/develop

### PR DESCRIPTION
**Pull request by RuslanPleskun**
Task #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,14 @@ __Programming language:__ JavaScript
 __Actions:__ Read, Watch, Buy  
 __Run project:__ https://ruslanpleskun.github.io/cstd-pleskunruslan-2122/  
 __Run tests:__ `npm test`  
-__Release version:__ ![version](https://img.shields.io/badge/version-1.0-blue)  
-__Build Status:__ [![BuildAndTest](https://github.com/RuslanPleskun/cstd-pleskunruslan-2122/actions/workflows/test.yml/badge.svg?branch=feature%2Fpleskunruslan%2Fdevelop&event=push)](https://github.com/RuslanPleskun/cstd-pleskunruslan-2122/actions/workflows/test.yml)  
+__GIT TAG:__ `Streaming_Service_v2.0_21133`
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md]: tag format should be:
image

Comment (RuslanPleskun) [reply README.md]: Fixed. Could you please check again whether it's correct?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ __Programming language:__ JavaScript
 __Actions:__ Read, Watch, Buy  
 __Run project:__ https://ruslanpleskun.github.io/cstd-pleskunruslan-2122/  
 __Run tests:__ `npm test`  
-__GIT TAG:__ `Streaming_Service_v2.0_21133`  
+__GIT TAG:__ `StreamingService_v1.1_14_21143`
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md]: should be: StreamingService_v1p1_WW21483

Comment (RuslanPleskun) [reply README.md]: Done

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**